noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/TweetGeotagNotRequired.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.cloudberry.noah.adm;
+
+import edu.uci.ics.cloudberry.gnosis.USGeoGnosis;
+import twitter4j.GeoLocation;
+import twitter4j.Status;
+
+public class TweetGeotagNotRequired {
```

Comment:
This function shared all the logics as the original one. Can you just change the 
`toADM(Status status, USGeoGnosis gnosis)`
to 
`toADM(Status status, USGeoGnosis gnosis, boolean requireGeoField)`  
to differentiate the behavior?